locales-src/build-locales.js

```diff
@@ -7,6 +7,7 @@ import extraAliases from "./extra_aliases.js"
 import scratch_l10n from "scratch-l10n"
 // We can't `import {default}` since it's a reserved word.
 const localeNames = scratch_l10n.default
+localeNames.en.name = "English";
```

Comment:
I suspect you'll want to update the built-in English locale here: https://github.com/scratchblocks/scratchblocks/blob/ca161ab3bd718d1e602f5c49faf2b1c8fdac25fd/syntax/blocks.js#L194

Reply:
Yes, that was right.